features/javascript/fields/method.feature

```diff
@@ -0,0 +1,47 @@
+@javascript @stable
```

Comment:
I'm wondering whether the `@stable` tag is good for anything... do you happen to know whether we use it anywhere?

Reply:
Gavel.js only runs those tests which are marked with the `@stable` tag. I believe it was meant to allow `@proposal` and `@draft` in the spec, without affecting the CI of the implementation (i.e. `gavel`).

Reply:
Ah. I don't think we need this kind of granularity now. Having proposals in Pull Requests seem okay for our needs. No priority in dropping the label though, up to you.

Reply:
`@stable` tag removed in 411239f34a86fe364ec202db9a60ebaf9381a3e9.

I've removed any mention of it, including the "add-implementation" guidelines. Those guidelines, however, look quite obsolete and I would propose to change/remove them as they do not depict the actual process of contributing to the spec anymore (no language-specific implementations, no tags).

Reply:
Agreed